packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx

```diff
@@ -72,6 +72,7 @@ export class InputMapFindPlace extends React.Component<
     private selectedIconUrl: string;
     private shouldFitBoundsIdx = 0;
     private currentBounds: [number, number, number, number] | undefined = undefined;
+    private renderGeocodeButton: boolean = false;
```

Comment:
as discussed earlier offline, this should be in the state instead of a private field

Reply:
@samuel-duhaime this is a react component, so anything that should have an impact on the rendering and display should be part of the state, as re-rendering on state change is part of the react internal mechanism. If it's a private field, here, you have to trigger somehow a re-render or just wait for one to magically happen (which happens quite often enough in our app, but still bad practice to rely on this).

In a React object component, you would put as private field some data that you calculate once (or a few times), that you want to keep between renders or use in many functions, but that does not have impact on the current display/rendering.

In this case, when the `renderGeocodeButton` becomes `true`, you _want_ the widget to be re-rendered to activate the button, so it goes in the state.